LayoutTests/fast/canvas/canvas-put-image-data-after-draw-expected.html

```diff
@@ -0,0 +1,4 @@
+<body style="margin: 0px">
+  <div style="width:300px; height: 150px; background-color: rgba(128, 128, 128, .5)""></div>
+</body>
+    
```

Comment:
To overcome the pixel differences on some ports, maybe the expected file should be like this:
```
<body style="margin:0">
<canvas id="a"></canvas>
<script>
    const imageData = new ImageData(300, 150);
    imageData.data.forEach((item, index) => {
        imageData.data[index] = 128;
    });
    const ctxA = a.getContext('2d');
    ctxA.putImageData(imageData, 0, 0);
</script>
</body>
```

Reply:
done